rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    repo:
+      # IRR REPOS ARE ADDED HERE
+      - rpc_maas:
+        properties:
+          - github:
+              url: https://github.com/rcbops/rpc-maas
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
```

Comment:
Ah, I see. trusty is the default image, so we only need to override the image for the ``xenial`` axis like so:

```
      - trusty
      - xenial:
          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
```